pyopencl/cl/pyopencl-complex.h

```diff
@@ -77,6 +77,13 @@
     ; \
   } \
   \
+  inline TP TPROOT##_fma(TP c, TP a, TP b)            \
+  { \
+    return TPROOT##_new( \
+        (c.real + a.real*b.real) - a.imag*b.imag, \
+        (c.imag + a.real*b.imag) + a.imag*b.real); \
+  } \
+  \
```

Comment:
That's sort of false advertisement. :) This doesn't use `fma` internally.

Reply:
I renamed to `c{double, float}_muladd` to not say that this is a fused multiply-add.

Reply:
That's not how I figured this might go. Why not just use the FMA function in OpenCL?